src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -174,6 +174,9 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
             ledgerObject = lastTwoObjects[0].second.empty()? std::make_optional(lastTwoObjects[1].second) : std::make_optional(lastTwoObjects[0].second);
             output.deleted_ledger_index = lastTwoObjects[0].second.empty()? lastTwoObjects[0].first : 0;
         } 
+        else {
+            return Error{Status{"entryExceedsExpected"}};
```

Comment:
this should be a log message instead of returning error error because this is a erroneous state, it doesn't make sense to return anything here

Reply:
remove this entire block should be fine